.codecov.yml

```diff
@@ -2,7 +2,7 @@
 
 codecov:
   notify:
-    after_n_builds: 27  # The number of test matrix+lint jobs uploading coverage
+    after_n_builds: 20  # The number of test matrix+lint jobs uploading coverage
```

Comment:
Or 23..

Reply:
Fixed in https://github.com/aio-libs/yarl/commit/43503cc8f068be45aba71a3f72cf237fad57a05b

Reply:
Setting fail_ci_on_error would also alert to any failed uploads immediately, without needing to try and keep this value in check. We originally set this value just to avoid getting email notifications from codecov before it had full coverage. Setting it too high just complicates maintenance, as observed when it took me a couple of hours before realising why codecov wasn't working.

Ideally, codecov would allow us some way to have the notification sent as soon as the test matrix completes.

Reply:
@Dreamsorcerer Yes, that option also needs to be set. But this specific setting is the way to allow notifications upon matrix completion, as it'd see all the uploads. If it's set lower, Codecov starts polling GH API too early, which consumes more rate limit and contributes to flakiness. Hence, the need for this to match for best stability possible.